pkgs/development/libraries/liblxi/default.nix

```
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, autoreconfHook
+, libtirpc, rpcsvc-proto, avahi, libxml2
+}:
+
+stdenv.mkDerivation {
```

It's not possible to access `version` in `fetchFromGitHub` otherwise.
```suggestion
stdenv.mkDerivation rec {
```